test_charging.py

```diff
@@ -0,0 +1,12 @@
+from charging import *
+import unittest
+
+class chargingTest(unittest.TestCase):
+    def test_range_and_count(self):
+        self.assertEqual(charging_measurement([4, 5]), ['4 - 5, 2'])
+        self.assertEqual(charging_measurement([3, 3, 5, 4, 10, 11, 12]), ['3 - 5, 4', '10 - 12, 3'])
+        self.assertEqual(charging_measurement([3]), ['3, 1'])
```

Comment:
Add a test for multiple identical readings. E.g., an input of [3,3,3,3] must give ['3, 4']. See if it works